apps/desktop-wallet/src/api/persistedCacheVersionStorage.ts

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+export type PersistedCacheVersion = string
```

Comment:
Nit: I would always specify that we're talking about the `PersistedQueryCache`, like you did in the related hooks and other functions.
We may persist other caches in the future.

Reply:
Done ✅